pkg/oauth/server.go

```diff
@@ -220,7 +220,7 @@ func (s *server) RegisterRoutes(server *web.Server) {
 		}),
 	)
 
-	csrfMiddleware := middleware.CSRF("_csrf", s.config.Mount, s.config.CSRFAuthKey)
+	csrfMiddleware := middleware.CSRF("_csrf", "/", s.config.CSRFAuthKey)
```

Comment:
Won't this now conflict with the console?

Reply:
The console uses `_console_csrf`.